parser.mly

```diff
@@ -7,6 +7,7 @@
 %token LPAREN RPAREN LBRACKET RBRACKET LBRACE RBRACE
 %token COLON EQUAL LEFTARROW TRIPLE_DOT COMMA
 %token CONST MUT BRANCH GOTO PRINT OSR STOP READ DROP CLEAR RETURN CALL VERSION FUNCTION
+%token ARRAY LENGTH
```

Comment:
I'll do the update, but only after we converge on a syntax and merge something, so that the design has stabilized.